### PR TITLE
fix: replace astronvim.utils.buffer (v3) with astrocore.buffer (v4) in map…

### PR DESCRIPTION
**Pull request by HalibutGitWiz**
The title says it all !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,11 +98,11 @@ local opts = {
       ["<C-s>"] = { ":w!<cr>", desc = "Save File" },
       -- navigate buffer tabs with `H` and `L`
       L = {
-        function() require("astronvim.utils.buffer").nav(vim.v.count > 0 and vim.v.count or 1) end,
+        function() require("astrocore.buffer").nav(vim.v.count > 0 and vim.v.count or 1) end,
```
Comment (mehalter) [inline README.md:101]:
```suggestion
        function() require("astrocore.buffer").nav(vim.v.count1) end,
```

File: README.md [hunk 2]
```diff
@@ -98,11 +98,11 @@ local opts = {
...
         desc = "Next buffer",
       },
       H = {
-        function() require("astronvim.utils.buffer").nav(-(vim.v.count > 0 and vim.v.count or 1)) end,
+        function() require("astrocore.buffer").nav(-(vim.v.count > 0 and vim.v.count or 1)) end,
```
Comment (mehalter) [inline README.md:105]:
```suggestion
        function() require("astrocore.buffer").nav(-vim.v.count1) end,
```

**Review by mehalter — CHANGES_REQUESTED** [review]
Thanks for spotting this! I recommended a couple other good code changes to simplify this if you could put that into this PR as well :)

**Comment by HalibutGitWiz** [comment]
Changes added (and tested in my own config, just in case)

**Pull request merged**